### DISCUSSION TITLE
Plausible with a Webapp

### DISCUSSION THREAD

**Discussion by FabiaxS** [plausible/analytics#2290] [Self-Hosted Support]
I have Django hosted with Nginx on DigitalOcean. Now I want to install Plausible Analytics. How do I do this? How do I change the Nginx config to get to the Plausible dashboard with mydomain/plausible for example?

**Comment by FarisZR** [comment]
Plausible AFAIK doesn't support being deployed to sub-dir.

Use https://caddyserver.com if you want simpler configs, and just configure a separate subdomain for Plausible.
